README.md

```diff
@@ -1,2 +1,202 @@
-# saritasa-github-actions
-GitHub actions
+## CI-wordpress reusable workflow
+
+### Deploying
+
+In order to deploy, you need to perform some steps:
```

Comment:
include videos you created (ru/en) here.

Reply:
I added links to the RU video, I will add the EN video a little later